server/scripts/yearOneImages.js

```diff
@@ -44,7 +44,7 @@ as = SongModel.Song.find()
   .populate('mainInstrument')
   .populate('secondaryInstrument')
   .then( results => {
-    const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+    const delay = () => new Promise(resolve => setTimeout(resolve, 2500));
```

Comment:
Can you fix this so that it allows ms to be set in the delay itself. Right now you've hardcoded it to 2.5 seconds. If you look below you'll see that it can be set in the function itself.